lib/phoenix_live_view/utils.ex

```diff
@@ -90,17 +90,28 @@ defmodule Phoenix.LiveView.Utils do
   @doc """
   Returns the socket's flash messages.
   """
-  def get_flash(%Socket{private: private}) do
-    private[:flash]
+  def get_flash(%Socket{assigns: assigns}), do: assigns.flash
```

Comment:
This function does not seem to be used outside of this module. :)

Reply:
It's now used in channel used after my latest revisions